packages/docsite/docs/golems/claude.md

```
@@ -52,14 +52,15 @@ Learn more: [link to blog post]
 Autonomous code improvements running at **4am daily**:
 
 ```bash
-bun src/night-shift/runner.ts --target songscript
+bun src/night-shift.ts
 ```
 
 **Per-repo sessions:**
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)